Workbooks/Workloads/Monitoring checklist/Monitoring checklist.workbook

```diff
@@ -0,0 +1,123 @@
+            "resourceType": "microsoft.resourcegraph/resources"
+          },
+          {
+            "id": "fbc8b0f8-bcb2-4ca7-938d-5578e966a7c2",
+            "version": "KqlParameterItem/1.0",
+            "name": "OnboardingStatus",
+            "label": "Onboarding status",
+            "type": 1,
+            "isRequired": true,
+            "query": "Resources\n| where id == '{VirtualMachine}'\n| project ['Virtual machine'] = tolower(id), Name = name, ['Resource group'] = strcat('/subscriptions/', subscriptionId, '/resourceGroups/', resourceGroup), Subscription = subscriptionId, Location = location,  OsType = iff(properties.osProfile.windowsConfiguration == '', 'Linux', 'Windows'), Offer = tostring(properties.storageProfile.imageReference.offer)\n//| where Offer == 'UbuntuServer' //and Location == 'centralus'\n| join kind = leftouter (Resources\n    | where type =~ 'microsoft.compute/virtualmachines/extensions' and name in~ ('omsagentforlinux', 'OMSExtension')\n    | project Vm = extract(@'(.+)/extensions/.+', 1, tolower(id)), WorkspaceId = tostring(properties.settings.workspaceId)\n    | where Vm != ''\n    ) on $left.['Virtual machine'] == $right.Vm\n| join kind = leftouter (Resources\n    | where type =~ 'microsoft.compute/virtualmachines/extensions' and name =~ 'workload.linuxconfigagent'\n    | project Vm = extract(@'(.+)/extensions/.+', 1, tolower(id)), Insight = tostring(tags[\"monitoring-insight\"])\n    | where Vm != ''\n    ) on $left.['Virtual machine'] == $right.Vm\n| project-away Vm, Vm1\n| join kind = leftouter (Resources\n| where type =~ 'microsoft.operationalinsights/workspaces'\n| project Workspace = id, location, WorkspaceId = tostring(properties.customerId)) on WorkspaceId\n| project-away ['Virtual machine'], ['Resource group'], Subscription, Insight, WorkspaceId1, location\n| project tostring(pack_all())\n",
+            "crossComponentResources": [
+              "value::all"
+            ],
+            "isHiddenWhenLocked": true,
+            "queryType": 1,
+            "resourceType": "microsoft.resourcegraph/resources"
+          },
+          {
+            "id": "77d59cac-0e77-4a4f-9857-8586aa7ca26e",
+            "version": "KqlParameterItem/1.0",
+            "name": "Text",
+            "type": 1,
+            "query": "Resources\n| take 1\n| project x = dynamic({OnboardingStatus})\n| project label = strcat('**Virtual machine: ', x.Name, '**',\n    '<br /><br />',\n    strcat(iff(x.Location in~ ('centralus', 'eastus', 'westus'), '✅', '❌'), ' Located in ', x.Location),\n    '<br /><br />',\n    strcat(iff(x.OsType =~ 'linux', '✅', '❌'), ' OS is ', x.OsType),\n    '<br /><br />',\n    strcat(iff(x.Offer =~ 'ubuntuserver', '✅', '❌'), ' OS offer is ', x.Offer),\n    '<br /><br />',\n    strcat(iff(x.WorkspaceId != '', '✅', '❌'), ' Configured with the OMS agent'),\n    '<br /><br />',\n    strcat(iff(x.Workspace != '', '✅', '❌'), ' Can access to the Log Analytics workspace')\n    )",

```

Comment:
How will we localize these strings in the query?

Reply:
We don't have a good story for localizating strings coming from queries. We could have it return constants and then use the criteria params with loc'd strings.